config/locales/en.yml

```
@@ -27,6 +27,7 @@ en:
   request_viewing: 'Request Viewing'
   request_form_success: 'Thank you! We have received your request and will be in contact shortly.'
   request_form_error: 'There was an problem sending your request. Please fix any errors and try again.'
+  text_this: 'Text this'
```

Good idea!  Should probably do the same for 'Email this' and 'Request this'
```suggestion
  text_this: 'Text this'
  request_this: 'Request this'
  email_this: 'Email this'
```
and then make all calls to this `t('text_this')`, `t('request_this')` or `t('email_this')`